includes/Classifai/Normalizer.php

```diff
@@ -49,6 +49,17 @@ public function normalize_content( $post_content, $post_title = '', $post_id = f
 		/* Replace HTML linebreaks with newlines */
 		$post_content = preg_replace( '#<br\s?/?>#', "\n\n", $post_content );
 
+		/**
+		 * Hook to filter post content before stripping HTML tags.
+		 *
+		 * @hook classifai_normalize_content_before_strip_all_tags
+		 *
+		 * @param {string} $post_content The post content.
+		 *
+		 * @return {string} The filtered Post content.
+		 */
+		$post_content = apply_filters( 'classifai_normalize_content_before_strip_all_tags', $post_content );
```

Comment:
We have a filter that runs after all normalization happens (`classifai_normalize`). I'd suggest instead of adding this filter prior to tags being stripped, let's add a filter at the very beginning of this method to allow modifying the content before any normalization happens. Maybe `classifai_pre_normalize` could be the filter name?